zookeeper-server/src/main/java/org/apache/zookeeper/server/DumbWatcher.java

```diff
@@ -69,7 +69,7 @@ public void sendCloseSession() { }
     void setSessionId(long sessionId) { }
 
     @Override
-    void sendBuffer(ByteBuffer closeConn) { }
+    void sendBuffer(ByteBuffer... closeConn) { }
```

Comment:
Why have you changed the serialization logic to use multiple buffers instead of a single one?